### ISSUE TITLE
focus background is black again

### ISSUE THREAD

**Issue by sgratzl** [cmu-delphi/www-covidcast#473]
**Actual Behavior:**

when I select a button using tab (as in screenreaders) it is again a black background:

![image](https://user-images.githubusercontent.com/4129778/94126674-3059f380-fe58-11ea-8fa7-db93f531e8c1.png)

**Steps to Reproduce the Bug:**

1. https://dev--cmu-delphi-covidcast.netlify.app/?sensor=doctor-visits-smoothed_adj_cli&level=county&date=20200918&signalType=value&encoding=color&mode=overview
1. use tabs a bunch of times till you get to the button

**Expected behavior**

back to the old behavior of no black background
